gsoc/2022/index.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Google Summer of Code"
+author: "Andrei Gheata & Benedikt Hegner"
+layout: default
+---
+
+# ![CERN]({{ site.baseurl }}/images/CERN-HSF-GSoC-logo.png){:height="100px"} Google Summer of Code 2023
+
+## Introduction
+
+[Google Summer of Code](https://summerofcode.withgoogle.com) is a program that allows students to contribute to the development of open-source projects, mentored by participating organizations.
+
+Particle physics is an exciting field where large collaborations of scientists collect
+and analyze petabytes of data from high-energy physics experiments, such as those at the Large Hadron Collider,
+hosted at the CERN laboratory in Geneva, Switzerland.
+Some of the questions that we collectively ask are:
+
+- what are the fundamental blocks that make up our Universe?
+- what is the nature of dark matter and dark energy?
+- what is the nature of the asymmetry between matter and antimatter?
+- what was early Universe like?
+
+To answer these questions, particle physicists build software to simulate and analyze what happens in particle physics detectors.
+
+The CERN software for experiments (CERN EP-SFT) group has participated in the GSoC since 2011. Since 2017 the program has expanded to involve the high-energy physics community under the umbrella of the HEP Software Foundation.
+
+Information from last year’s GSoC can be found [here]({{ site.baseurl }}/gsoc/2021/index.html).
+
+
+### GSoC 2022
+
+In 2022 HSF is participating in the program as a GSoC umbrella organization, under the name CERN-HSF. The HSF project ideas are listed [below](#projects-in-2022). There are important changes in the [program rules](https://summerofcode.withgoogle.com/rules/)  and the [timeline](https://summerofcode.withgoogle.com/how-it-works/) compared to the previous year, described in this news [blog](https://opensource.googleblog.com/2021/11/expanding-google-summer-of-code-in-2022.html) and in the GSoC 2022 [announcement](https://groups.google.com/g/google-summer-of-code-mentors-list/c/zIX5CEOJWZo/m/94V-2F1FBgAJ). Most notably, there are now both 175 hours and 350 hours projects. The maximum duration of projects was now extended to 22 weeks, which gives some more flexibility on the scheduling of work.
```

Comment:
The link to the GSoC 2022 announcement seems to be dead (or only available to subscribers to the list). We can probably just remove it now.